querydsl-sql/src/main/java/com/querydsl/sql/Union.java

```diff
@@ -25,23 +25,15 @@
  *
  * @param <RT> return type of projection
  */
-public interface Union<RT> extends SubQueryExpression<RT> {
+public interface Union<RT> extends SubQueryExpression<RT>, Fetchable<RT> {
 
     /**
-     * Get the projection as a typed List
-     *
-     * @return results
+     * Use fetch instead
```

Comment:
You might want to change this to `@deprecated Use {@link Union#fetch()}` so people can easily navigate to the new method.